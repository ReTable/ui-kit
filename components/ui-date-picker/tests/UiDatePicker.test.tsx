import { create } from 'react-test-renderer';
import { describe, expect, it } from 'vitest';

import { UiDatePicker } from '~';

describe('UiDatePicker', () => {
  it('works', () => {
    const tree = create(<UiDatePicker />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
