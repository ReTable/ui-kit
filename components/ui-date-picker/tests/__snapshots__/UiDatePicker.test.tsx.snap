// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UiDatePicker > works 1`] = `
<div
  className="tabula_ui_date_picker__1uj5ens0 "
/>
`;
